build/lib/packager.js

```diff
@@ -85,7 +83,7 @@ function determineBabelOptions() {
 	const chromeVersion = parseInt(found[1] + found[2]); // concat the first two numbers as string, then turn to int
 	// Now pull out min IOS target
 	// eslint-disable-next-line security/detect-non-literal-require
-	const minSupportedIosSdk = version.parseMin(require(path.join(ROOT_DIR, 'iphone/package.json')).vendorDependencies['ios sdk']);
+	const minSupportedIosSdk = require(path.join(ROOT_DIR, 'iphone/package.json')).minIosVersion;
```

Comment:
@ewanharris Can you please change const  name 'minSupportedIosSdk' to 'minSupportedIosVersion' because it is a bit confusing.